mkdocs/docs/configuration.md

```diff
@@ -22,35 +22,11 @@ hide:
   - under the License.
   -->
 
-# Catalogs
-
-PyIceberg currently has native support for REST, SQL, Hive, Glue and DynamoDB.
-
-There are three ways to pass in configuration:
-
-- Using the `~/.pyiceberg.yaml` configuration file
-- Through environment variables
-- By passing in credentials through the CLI or the Python API
-
-The configuration file is recommended since that's the easiest way to manage the credentials.
-
-Another option is through environment variables:
-
-```sh
-export PYICEBERG_CATALOG__DEFAULT__URI=thrift://localhost:9083
-export PYICEBERG_CATALOG__DEFAULT__S3__ACCESS_KEY_ID=username
-export PYICEBERG_CATALOG__DEFAULT__S3__SECRET_ACCESS_KEY=password
-```
-
-The environment variable picked up by Iceberg starts with `PYICEBERG_` and then follows the yaml structure below, where a double underscore `__` represents a nested field, and the underscore `_` is converted into a dash `-`.
-
-For example, `PYICEBERG_CATALOG__DEFAULT__S3__ACCESS_KEY_ID`, sets `s3.access-key-id` on the `default` catalog.
-
-# Tables
+## Tables
```

Comment:
Adding the extra `#` so that the `Table of contents` sidebar can be rendered correctly:
![Screenshot 2024-07-14 at 02 15 54](https://github.com/user-attachments/assets/e358a404-091b-4ad0-9b46-f15a7fa5ba55)